projects/indicators/src/modules/chevron/chevron.component.ts

```diff
@@ -6,15 +6,40 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
   templateUrl: './chevron.component.html',
 })
 export class SkyChevronComponent {
-  @Output()
-  public directionChange = new EventEmitter<string>();
+  @Input()
+  public ariaControls: string;
 
   @Input()
```

Comment:
Does this need a JSDocs comment to add the public property to the docs? I assume so. Based on a scan of the existing docs for existing `label` and `ariaLabel` properties, we usually use some shared text that we flesh out with details for the particular instance. I'm not sure what details we need to add here, but I assume we should use something like this as our starting point:

```Specifies an ARIA label for the chevron. This sets the chevron's aria-label attribute to support accessibility.```